g2p/mappings/__init__.py

```diff
@@ -432,8 +432,11 @@ def load_mapping_config_from_path(path_to_mapping_config: Union[str, Path]):
                     mapping["parent_dir"] = parent_dir
         try:
             return MappingConfig(**loaded_config)
+        except exceptions.MalformedMapping as e:
+            e.message = f"{e.message}\nProblem in config file: {path}"
+            raise e
```

Comment:
This code is currently unreachable (we never raise MalformedMapping now that we're using Pydantic) but I'm going to leave it here because I want to figure out, for #320, how to catch the pydantic exception and raise a MalformedMapping from it to full it up here with a message telling the user which file the problem occurs in, with the pydantic error saying which field is the problem.

Reply:
agreed - we can improve on these. thanks for raising the issue @joanise !